HW4/task2.js

```diff
@@ -0,0 +1,33 @@
+/*
+2. Задача на фільтрування масиву
+
+Реалізуйте фільтрування імен які починаються з голосної двома способами:
+
+через умовну конструкцію
+через вбудований метод масивів
+
+
+Рішення має працювати незалежно від конкретних значень в масиві імен
+ */
+// тут ваш код...
+
+const userNames = ['Петро', 'Емма', 'Юстин', 'Ілля', 'Марта', 'Яна', 'Василь', 'Антон', 'Олена'];
+let filteredNames;
+filteredNames = [];
```

Comment:
```suggestion
let filteredNames = [];
```
Тут можна зразу й об'явити змінну та ініціювати її пустим масивом